MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include LICENSE
+include OWNERS
+include *.md
+include *.txt
+include *.ini
+exclude .gitignore
+exclude .gitreview
+
+global-exclude *.pyc
```

Comment:
nit: add new line at the end of the file.

Reply:
Done